lib/grape/all/grape.rbi

```diff
@@ -0,0 +1,224 @@
+    positive_number: nil,
+    acord130_question_number: nil,
+    id_for_entity_type: nil,
+    &block)
+  end
+
+  sig do
+    params(
+      name: Symbol,
+      type: T.untyped,
+      allow_blank: T::Boolean,
+      values: T::Array[T.untyped],
+      date_parameter: T::Boolean,
+      positive_number: T::Boolean,
+      acord130_question_number: T::Boolean,
+      percent_parameter: T::Boolean,
+      id_for_entity_type: T.untyped,
+      desc: String
+    ).void
+  end
+  def self.optional(
+    name,
+    type: String,
+    desc: nil,
+    allow_blank: nil,
+    values: nil,
+    date_parameter: nil,
+    positive_number: nil,
+    acord130_question_number: nil,
+    percent_parameter: nil,
+    id_for_entity_type: nil,
+    &block
+    )
+  end
+
+  sig do
+    params(
+      block: T.nilable(T.proc.void)
+    ).returns(Hash)
+  end
+  def self.params(&block)
+  end
+
+  sig do
+    params(
+      route: String,
+      block: T.nilable(T.proc.void)
+    ).void
+  end
+  def self.put(route, &block)
+  end
+
+  sig do
+    params(
+      conditional: Symbol,
+      block: T.nilable(T.proc.void)
+    ).void
+  end
+  def self.given(conditional, &block)
+  end
+
+
+  sig do
+    params(
+      block: T.nilable(T.proc.void)
+    ).void
+  end
+  def self.helpers(&block)
+  end
+
+  sig do
+    params(
+      route: String,
+      block: T.nilable(T.proc.void)
+    ).void
+  end
+  def self.post(route, &block)
+  end
+
+  sig do
+    params(
+      route: String,
+      block: T.nilable(T.proc.void)
+    ).void
+  end
+  def self.delete(route, &block)
+  end
+
+  sig do
+    params(
+      route_param_name: T.any(String, Symbol),
+      block: T.nilable(T.proc.void)
+    ).void
+  end
+  def self.route_param(route_param_name, &block)
+  end
+
+  sig do
+    params(
+      route: String,
+      block: T.nilable(T.proc.void)
+    ).void
+  end
+  def self.get(route = nil, &block)
+  end
+
+  sig do
+    params(
+      space: T.any(T.nilable(Symbol), T.nilable(String)),
+      options: T.nilable(Hash)
+    ).void
+  end
+  def self.resource(space = nil, options = {})
+  end
+
+  sig do
+    params(
+      space: T.any(T.nilable(Symbol), T.nilable(String)),
+      options: T.nilable(Hash)
+    ).void
+  end
+  def self.namespace(space = nil, options = {})
+  end
+
+
+  sig do
+    params(
+      space: T.any(T.nilable(Symbol), T.nilable(String)),
+      options: T.any(
+        T.nilable(Hash),
+        String,
+      )
+    ).void
+  end
+  def self.route(space = nil, options = {})
+  end
+
+  sig do
+    params(
+      space: T.any(T.nilable(Symbol), T.nilable(String)),
+      options: T.nilable(Hash)
+    ).void
+  end
+  def self.resources(space = nil, options = {})
+  end
+
+  sig do
+    params(
+      api_class: T.any(
+        T.class_of(Grape::API),
+        Hash
+      )
+    ).void
+  end
+  def self.mount(api_class)
+  end
+
+  sig do
+    params(
+      object: T.untyped,
+      with: T.nilable(T.class_of(Grape::Entity))
```

Comment:
i think this line will crash if you don't have the https://github.com/ruby-grape/grape-entity gem installed